lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -549,9 +549,25 @@ states:
         checkIfDisabled:
           f2f:
             targetState: MULTIPLE_DOC_CHECK_PAGE
+        checkFeatureFlag:
+          f2fVolumesJourneyEnabled:
+            targetState: PROVE_ANOTHER_WAY_J2
       alternate-doc-invalid-passport:
         targetState: MITIGATION_05_OPTIONS
 
+  PROVE_ANOTHER_WAY_J2:
+    response:
+      type: page
+      pageId: prove-identity-another-type-photo-id
+      context: passport
+    events:
+      otherPhotoId:
+        targetState: CRI_DRIVING_LICENCE_J3
+      f2f:
+        targetState: F2F_PYI_POST_OFFICE
+      returnToRp:
+        targetState: RETURN_TO_RP
```

Comment:
These returnToRp events will need to go via TICF before sending the user back to the service. I think you can use the `INELIGIBLE/INELIGIBLE_SKIP_MESSAGE` state for where a user doesn't have a CI